config/src/main/java/com/alibaba/nacos/config/server/service/DynamicDataSource.java

```diff
@@ -47,7 +47,7 @@ public ApplicationContext getApplicationContext() {
     public DataSourceService getDataSource() {
         DataSourceService dataSourceService = null;
 
-        if (STANDALONE_MODE && !propertyUtil.isStandaloneUseMysql()) {
+        if (STANDALONE_MODE && !PropertyUtil.isStandaloneUseMysql()) {
```

Comment:
Then the `propertyUtil` member variable (line 35) could be removed?

Reply:
ok